video/out/opengl/context_angle.c

```diff
@@ -203,14 +239,28 @@ static int angle_init(struct MPGLContext *ctx, int flags)
         goto fail;
     }
 
-    EGLint d3d_types[] = {EGL_PLATFORM_ANGLE_TYPE_D3D11_ANGLE,
-                          EGL_PLATFORM_ANGLE_TYPE_D3D9_ANGLE};
+    struct {
+        EGLint platform, device_type;
+    } d3d_types[] = {
```

Comment:
Fair enough. I wonder if it would be better as an array of EGLint[], since each attribute is specified in each display type anyway.